reconfigure graph and graph edge content (#533)
Macrometacorp/docs: docs/graphs/getting-started.md

---

> title: Getting started with Graphs

Reviewer: ```suggestion
title: Getting Started with Graphs
```

sidebar_position: 10
---

> In this getting started we will create a document collection, a edge collection, and a graph that creates relationships between the collections. We will then query that graph.

Reviewer: ```suggestion
This page shows you how to create a document collection, a graph edge collection, and a graph that creates relationships between the collections. You will then query that graph.
```

    INSERT edge IN flights
```

## Create a Graphs

Create a graph named airline with and edges in flights and both from and to vertices in cities.

## Query your Graph

> Storing (and retrieving) a graph is one thing, but the actual problems only begin when we want to query information about a graph.

Reviewer: Generally in technical documentation, we write for "you," because "we" is ambiguous and might be confusing.
